File: code_15_11_28.rb
# Alan Badillo Salas | badillo.soft@hotmail.com
# Ruby - Números primos menores o iguales a 100

prs = []
2.upto(100) do |n| 
	prs[n] = true if prs[n].nil?
	
	(2 * n).step(100, n) do |k|
		prs[k] = false
	end
	
	puts "#{n}\t#{prs[n] ? 'primo' : 'no'}"
end

File: code_15_11_28.py
# Alan Badillo Salas | badillo.soft@hotmail.com
# Python - Números primos menores o iguales a 100

prs = {}
for n in range(2, 101):
	if not prs.has_key(n):
		prs[n] = True
		
	for k in range(2 * n, 101, n):
		prs[k] = False
		
	print "%d\t%s" %(n, 'primo' if prs[n] else 'no')